cluster-scope/base/config.openshift.io/apiserver/cluster/apiserver.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    oauth-apiserver.openshift.io/secure-token-storage: "true"
+    release.openshift.io/create-only: "true"
+  name: cluster
```

Comment:
@chrisstafford I think the issue is this resource in the infra cluster: [https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/cluster/config.openshift.io~v1~APIServer/cluster/yaml](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/cluster/config.openshift.io~v1~APIServer/cluster/yaml)
It has the same name as the `APIserver` you're adding here. It's in the cluster but it's not in the repo(as far as I can tell) so maybe it was added via the console already. 

I think by removing just this file and just patching the existing resource will fix the issue but maybe the original resource should be deleted to add this `APIserver` resource

Reply:
Hmmm, I believe the APIServer is created on install, looking at the operatefirst cluster repo I only see a patch file for the APIServer and nothing in the base. I think the move here would be to remove the base file and edit the existing resource.

Reply:
@chrisstafford did this resolve the issue?